protos/sources/vega/snapshot/v1/snapshot.proto

```diff
@@ -697,13 +697,13 @@ message PartyOrders {
 
 message LiquidityPartiesOrders {
   repeated PartyOrders party_orders = 1;
-  // Required for key generation
+  // ID of the market these orders apply to. Required for key generation
```

Comment:
Should we say something like "Market ID of the orders....."

this matches the format of other comments where it starts with XXXX ID...